src/olympia/blocklist/tests/test_mlbf.py

```diff
@@ -98,7 +98,7 @@ def test_all_guids(self):
     def test_get_blocked_versions(self):
         blocked_versions = MLBF.get_blocked_versions()
         blocked_guids = blocked_versions.values()
-        assert len(blocked_guids) == 5
+        assert len(blocked_guids) == 5, blocked_guids
```

Comment:
A drive-by change because this is an intermittent ci failure and it'd be useful to know what the guids where when it failed.